lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -69,14 +69,14 @@
           lang="${course_overview.language}"
         % endif
       >
-<article class="course${mode_class}" aria-labelledby="course-title-${enrollment.course_id}" id="course-card-${course_card_index}">
+<article class="course${mode_class}" aria-labelledby="course-title-${enrollment.course.id}" id="course-card-${course_card_index}">
```

Comment:
I think this one works better. We already have course_overview in the context, right?

```suggestion
<article class="course${mode_class}" aria-labelledby="course-title-${course_overview.id}" id="course-card-${course_card_index}">
```

Reply:
It's actually the same, the value is copied from enrollment. But this will make the code cleaner anyway. Thank you @OmarIthawi . Done